text/chapter5.md

```diff
@@ -275,7 +280,7 @@ Failed pattern match
 
 Functions which return a value for any combination of inputs are called _total_ functions, and functions which do not are called _partial_.
 
-It is generally considered better to define total functions where possible. If it is known that a function does not return a result for some valid set of inputs, it is usually better to return a value with type `Maybe a` for some `a`, using `Nothing` to indicate failure. This way, the presence or absence of a value can be indicated in a type-safe way.
+It is generally considered better to define total functions where possible. If it is known that a function does not return a result for some valid set of inputs, it is usually better to return a value capable of indicating failure, such as type `Maybe a` for some `a`, using `Nothing` when it cannot return a valid result. This way, the presence or absence of a value can be indicated in a type-safe way.
```

Comment:
So I for one have been bit by overusing `Maybe` and swallowing errors that become very, very difficult to diagnose.  I thought maybe softening the language would be nice so that `Maybe` was just one suggested approach instead of saying that `Maybe` is _the_ approach in functional programming.